.github/kics-config.yml

```diff
@@ -0,0 +1,37 @@
+exclude-queries:
+  # query IDs can be found here: https://docs.kics.io/latest/queries/all-queries/
```

Comment:
I would suggest for future self when revisiting this pipeline to add in the comment also the reason why we exclude the specific rule

Reply:
i will add a comment stating why those queries are excluded